backstop/tests/header.html

```diff
@@ -104,108 +106,150 @@ <h1>Header</h1>
-                  aria-hidden="true"
-                ></svg-caret-down-small>
-              </button>
-
-              <svg-chevron-right
-                class="iui-chevron"
-                aria-hidden="true"
-              ></svg-chevron-right>
-
-              <!-- IModel Button -->
-              <button
-                class="iui-button iui-borderless iui-header-button iui-header-dropdown-button"
-                type="button"
-                aria-label="Model picker"
-                aria-expanded="false"
-              >
-                <svg-model
-                  class="iui-button-icon iui-header-button-icon"
+            <nav
+              class="iui-header-breadcrumbs"
+              aria-label="Breadcrumbs"
+            >
+              <ol class="iui-breadcrumbs-list">
+                <li class="iui-breadcrumbs-item">
+                  <span class="iui-header-breadcrumb-button-container">
+                    <button
+                      class="iui-header-breadcrumb-button"
+                      type="button"
+                      aria-label="Project picker"
+                      aria-expanded="false"
+                      id="test-button-1"
+                    >
+                      <img
+                        class="iui-header-breadcrumb-button-icon"
+                        src="../assets/image-test.jpg"
+                        draggable="false"
+                      />
+                      <span class="iui-header-breadcrumb-button-label">
+                        <div class="iui-header-breadcrumb-button-label-label">Project Alpha</div>
+                        <div class="iui-header-breadcrumb-button-label-sublabel">0x0123456789</div>
+                      </span>
+                      <svg-caret-down-small
+                        class="iui-header-breadcrumb-button-dropdown-icon"
+                        aria-hidden="true"
+                      ></svg-caret-down-small>
+                    </button>
+                  </span>
+                </li>
+                <li
+                  class="iui-breadcrumbs-separator"
                   aria-hidden="true"
-                ></svg-model>
-                <span class="iui-button-label">
-                  <div>Model Beta</div>
-                  <div class="iui-description">0x0123456789</div>
-                </span>
-                <svg-caret-down-small
-                  class="iui-button-icon"
+                >
+                  <svg-chevron-right aria-hidden="true"></svg-chevron-right>
+                </li>
+                <li class="iui-breadcrumbs-item">
+                  <span class="iui-header-breadcrumb-button-container">
+                    <button
+                      class="iui-header-breadcrumb-button"
+                      type="button"
+                      aria-label="Model picker"
+                      aria-expanded="false"
+                    >
+                      <svg-model
```

Comment:
In test images I see this images background is different. expected?

Reply:
Yes, that was intentional.  This passes our contrast scores where previously it did not.